Tests/Helper/LocaleHelperTest.php

```diff
@@ -35,6 +35,7 @@ public function testLanguage()
         $helper = $this->getHelper();
         $this->assertEquals('français', $helper->language('fr'));
         $this->assertEquals('français', $helper->language('fr_FR'));
+        $this->assertEquals('anglais américain', $helper->language('en_US'));
```

Comment:
yes, but "fr" and "fr_FR" always give the same result. This is why I tested with "en_US", because it gives a different result than "en".